controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -890,7 +896,10 @@ func (r *BareMetalHostReconciler) actionProvisioning(prov provisioner.Provisione
 // fields of a host.
 func clearHostProvisioningSettings(host *metal3v1alpha1.BareMetalHost) {
 	host.Status.Provisioning.RootDeviceHints = nil
-	host.Status.Provisioning.RAID = nil
+	// Only clear software raid volumes
+	if host.Status.Provisioning.RAID != nil {
+		host.Status.Provisioning.RAID.SoftwareRAIDVolumes = nil
+	}
```

Comment:
What's the reason to cleanup only the software RAID here?

Reply:
Because the hardware raid cannot be modified by the operating system, I think it can be retained and reused.

Reply:
Done.